common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -867,6 +867,23 @@ function (VideoPlayer, VideoStorage, i18n) {
 
         return true;
     }
+
```

Comment:
WIP function checks for the overlay captions cookie on page load and if present, automatically enables the captions for every video. If captions are turned off, the cookie is removed.